lib/internal/Magento/Framework/Session/SessionManager.php

```diff
@@ -200,6 +205,33 @@ public function start()
         return $this;
     }
 
+    /**
+     * Start session.
+     *
+     * @throws \Magento\Framework\Exception\SessionException
+     */
+    private function sessionStart()
```

Comment:
sessionStart accesses the super-global variable $_SESSION.
https://app.codacy.com/app/shyamranpara/magento2/pullRequest?prid=1494337

Reply:
@larsroettig, thank you for the review.
The problem has been fixed.